tests/operators/test_agnostic_aggregate_check.py

```diff
@@ -65,7 +65,7 @@ def setUpClass(cls):
         cls.aggregate_table_bigquery = Table(
             "aggregate_check_test",
             conn_id="bigquery",
-            schema="tmp_astro",
+            schema="ASTROFLOW_CI",
```

Comment:
We can use this opportunity to remove the hard-coded schema from (at least the python) tests - so people from outside Astronomer who may not have the ability to create this schema may still run the tests.